lib/t/feeds.ex

```diff
@@ -85,23 +85,46 @@ defmodule T.Feeds do
   ### Feed
 
   @type feed_cursor :: DateTime.t()
-  @type feed_profile :: {%FeedProfile{}, distance_km :: non_neg_integer}
+  @type feed_profile :: %FeedProfile{}
```

Comment:
This is confusing, feed profile is feed profile. I try to pick different names for composite types and not name the composite type by one of its elements.